walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -71,23 +71,23 @@ The Fuse Aggregation App provides information about flights. You deploy it using
 
 . Select the option to *Build and Deploy Online* in the *Target Environment* step.
 
-. Select the *Authorize* button to allow Launcher create resources in the OpenShift cluster. This triggers an OAuth flow, and eventually redirects back to the {launcher-service} wizard.
+. Click the *Authorize* button to allow Launcher create resources in the OpenShift cluster. This triggers an OAuth flow, and eventually redirects back to the {launcher-service} wizard.
```

Comment:
I would drop the ','.

A similar sentence below contains a *,* "This triggers an OAuth flow and eventually redirects back to the {launcher-service} wizard."